### DISCUSSION TITLE
VACCINATIONS: two-doses and one-dose

### DISCUSSION THREAD

**Discussion by lucasrodes** [owid/covid-19-data#2083] [General]
This is a centralized issue to discuss how to jointly handle two-dose and one-dose vaccines in the data. 

As of 2021-01-12, all approved vaccinations are administered in two doses. However, some future vaccines are announced to require only one dose.

A first proposal could be as in [England.csv](https://github.com/owid/covid-19-data/blob/master/scripts/scripts/vaccinations/automations/output/England.csv), where a new column `people_fully_vaccinated` has been introduced. Therefore, when analyzing all doses jointly (from all providers), we might prefer to use this column instead of `total_vaccinations`.

#### 2 Doses
Pfizer/BionTech, Moderna, AstraZeneca/Oxford university, Novavax, Medicago/GSK, CureVac, Gamaleya, Sinopharm I, Sinopharm II, SinoVac, Bharat Biotech

#### 1 Dose
Johnson & Johnson, CanSino

#### Unclear
ZFSW

Reference: [_Covid vaccine tracker: The shots available and the doses administered_, Financial Times, January 4 2021](https://www.ft.com/content/ac5e5ef8-bccb-482b-9f8d-0dab5cac6f9a) (accessed on January 12, 2021)

**Comment by edomt** [comment]
As always thank you @lucasrodes!

The [vaccination README](https://github.com/owid/covid-19-data/tree/master/public/data/vaccinations) now includes a detailed explanation of how our 3 metrics (`total_vaccinations`, `people_vaccinated`, `people_fully_vaccinated`) should be understood. I'll add here, for the sake of clarity, that `total_vaccinations` really means "total doses administered" and this would have been a better label for the variable, but we now have too many people relying on the file for external use to change that (and `total_vaccinations` is still a correct label, just not the most accurate).

Regarding calculations and data processing, I expect that things will get messy when some countries start mixing 1-dose vaccines and 2-dose vaccines in their vaccination program, but won't necessarily release the right data for us to calculate our metrics correctly.

For now, we have this data for [the UK](https://github.com/owid/covid-19-data/blob/master/public/data/vaccinations/country_data/United%20Kingdom.csv) (and England/Scotland/NI/Wales), and [Israel](https://github.com/owid/covid-19-data/blob/master/public/data/vaccinations/country_data/Israel.csv).

**Comment by lucasrodes** [comment]
Hi @edomt,
Thanks for your clarifications 🔝!

> The vaccination README now includes a detailed explanation of how our 3 metrics (total_vaccinations, people_vaccinated, people_fully_vaccinated) should be understood. 

Had a look before, great job!

> [...] but we now have too many people relying on the file for external use to change that (and `total_vaccinations` is still a correct label, just not the most accurate).
 
Couldn't agree more with this! Backwards compatibility is a must at this point

---

Having said this, for now, I guess this is still a problem for the future, once one-dose vaccines are in use. This - the dosage variability in different vaccines - came to my mind today and didn't want to forget, so I noted it down as an issue 😄.

**Comment by yoch** [comment]
Question: How to show the plot of `people_vaccinated` | `people_fully_vaccinated` on the [website](https://ourworldindata.org/covid-vaccinations) ?

**Comment by edomt** [comment]
@yoch The data explorer (the very big graph at the top of the page) only includes doses. But if you scroll down, you'll find a section **"What share of the population has received at least one dose of the COVID-19 vaccine?"** with data on people vaccinated, and **"What share of the population has been fully vaccinated against COVID-19?"** on people fully vaccinated.

**Comment by yoch** [comment]
Okay, thank you !

IMHO the section "What share of the population has received at least one dose of the COVID-19 vaccine?" is more relevant and deserve to be the main picture, because "doses administered per 100 people" introduce some confusion, especially when comparing with one doses vaccine in future.

**Comment by mikesherov** [comment]
> Okay, thank you !
> 
> IMHO the section "What share of the population has received at least one dose of the COVID-19 vaccine?" is more relevant and deserve to be the main picture, because "doses administered per 100 people" introduce some confusion, especially when comparing with one doses vaccine in future.

Came here to say this. The top chart on the website is doses, and framed "as per 100 people". This sets up an expectation that when we get to 100, we're done. But this really means we need to get to 200! So the rollout is actually half as fast as we'd like.

Deciding what the main graph is on the page is important and I humbly recommend it be either updated to "number of people receiving at least one dose" or "number of people fully vaccinated" so that 100 is the real target... not something somewhere between 100 and 200. See here for a Twitter thread on the confusion: https://twitter.com/mikesherov/status/1354103902875099138?s=21

thanks for the great work you all do and for the openness of this project!

**Comment by edomt** [comment]
Thanks for the feedback @yoch & @mikesherov 

Our choice to show total doses at the top of the page is a deliberate one (our team actually spends a fair amount of time discussing & debating how to ideally present information on each of our COVID-19 page).

I recently wrote a Twitter thread [here](https://twitter.com/redouad/status/1351626595186995202/) explaining our rationale for _currently_ (i.e. not necessarily permanently) preferring total doses: 

> Why do we mostly present vaccinations by number of doses, and not by number of people vaccinated (at least 1 dose) or number of people fully vaccinated (2 doses)? Because currently, it is still the most universal indicator to express the overall vaccination effort of a country.

> If we only measure people vaccinated (those with at least 1 dose), then some countries that are currently only giving second doses—such as Oman—will look like they're completely stagnating, when they're actually administering many doses.

> If we only measure people fully vaccinated (those with 2 doses), then some countries that have chosen to give many first doses before giving second doses—such as Denmark—will look like they're completely stagnating, when they're actually administering many doses.

> Note that all 3 indicators (doses, people vaccinated, people fully vaccinated) are crucial to understanding the speed and success of vaccination campaigns. And that's why we present all three, both in absolute numbers and per capita, on our site.

> But for now, in a situation where many countries started their vaccination on different schedules, and some countries are taking different paths on the issue of second doses, the number of doses administered still provides the clearest and most relevant overview of the situation.

**Comment by mikesherov** [comment]
While I appreciate the rationale about accurately representing velocity of roll out, it still doesn't serve as a foundation that contains a stable denominator to the ratio. "Per 100 people" will continue to belie the notion of progress. I believe most people will want to know actual progress towards "everyone has been vaccinated" or "herd immunity achieved". The natural understanding of 100 as the denominator expressing "percent of the way there" is inescapable, and will continue to mislead visitors. I suggest usability testing this because it's really counter to natural user expectation, even with the warning blurb.

**Comment by edomt** [comment]
Hi @mikesherov 

I understand your concern, and thank you for the time you took to voice it. **As I said above, this is the _current_ situation, and we may decide to change the layout of the page and reorder the charts.** In particular, "people fully vaccinated" may become the main metric we want to highlight in the future, both from a public health point of view and in terms of public interest.

On the other hand, we do get hundreds of messages per day that give us feedback on our work, both directly on the site (through our feedback button in the bottom-right corner of each page) and outside (Twitter, GitHub). And more than 50% of this feedback is currently about our vaccination page. When something looks wrong or is misinterpreted by a lot of our users, we almost instantly receive dozens of questions about it, and we act quickly to correct or clarify it. This is simply not the case about this issue.

Again, we're sincerely interested in user feedback and over the last 12 months, we've made hundreds of changes to our COVID-19 pages & charts based on it. But there's very little evidence for the confusion you're describing here, and it seems that the chart's subtitle is currently enough to help people understand how this metric works.

Edouard

**Comment by yoch** [comment]
Thanks for the answer, the rationale is quite convincing. But maybe it need to be more explicit (I don't know exactly how).

I've seen on Twitter that even good scientists can be confused by the presentation of results. You can see for example: https://twitter.com/EricTopol/status/1354182091148152833

**Comment by MichaelLLi** [comment]
Would it be possible to create another column indicating the total number of people with 1+ doses? This is e.g. recorded on the CDC website (https://covid.cdc.gov/covid-data-tracker/#vaccinations). 

The benefit of such recording is that it gives enough information to back-out the number of people receiving the 1-dose vaccine, which currently appears to have a significant efficacy difference with most two-dose vaccines. Currently as defined, the J&J vaccine would count towards both the columns "people_fully_vaccinated" and "people_vaccinated", making the statistics unidentifiable.

**Comment by lucasrodes** [comment]
@MichaelLLi Good point! Just to check If I understood it correctly (which I am not sure), does your proposal match the example below?

### Currently we have:
```
total_vaccinations = people_vaccinated + people_fully_vaccinated
```

### New column addition:
```
total_vaccinations = people_vaccinated + people_fully_vaccinated - people_fully_vaccinated_1_dose
```

where `people_fully_vaccinated_1_dose` would be the new column accounting for people fully vaccinated with J&J vaccine.

Not needed, but could also have a `people_fully_vaccinated_2_doses` (`people_fully_vaccinated` - `people_fully_vaccinated_1_dose`) column.

**Comment by edomt** [comment]
I'm actually not sure if I fully understand your suggestion @MichaelLLi: what's given by the US CDC is "Number of People Receiving 1 or More Doses", which is already what our "people_vaccinated" variable is.

**Comment by philipschellekens** [comment]
Thanks to the OWID team for the incredible efforts on this! 

Here's a suggestion on how to reconcile the various tensions between doses and people as well as the 100 vs 200% issue, which I am thinking of implementing on the [site](https://pandem-ic.com) I am running on the pandemic by World Bank income classification. 

### Data availability issues vs the need for a global indicator of coverage 

Countries report very differently on their vaccination efforts. In the OWID dataset we can distinguish between three cases:

1. countries that report only `total_vaccinations` (e.g. Saudi Arabia and UAE)
2. countries that report `total_vaccinations` and `people_vaccinated` (e.g. Bahrain and Bangladesh)
3. countries that report `total_vaccinations`, `people_vaccinated` and `people_fully_vaccinated` (e.g. India and the US).

One observation is that It's curious that for the countries under (2) it turns out that `total_vaccinations = people_vaccinated`, suggesting that the doses administered thus far have all been single shots.  Obviously, in the absence of further info, we don't know whether these shots belong to a single- or double-shot protocol. The countries under (1) do not report `people_vaccinated` opening up many possibilities about the number of people covered.  For the countries under (3), we do have a way to get an accurate picture.

In light of this, I agree with the earlier statement in this thread that it makes sense (for now) to focus on `total_vaccinations / population` as a way to represent vaccination progress. If it is the objective to show global vaccination progress, then the alternatives of `people_vaccinated `and `people_fully_vaccinated` unfortunately do not make sense because the data series is not globally comprehensive.   (Obviously, if the intention is to show a few individual countries, then that need not be true.) 

### Single- and double-protocol equivalents

One drawback as noted by @mikesherov is that full coverage is achieved at 200% as opposed to 100%. As Edouard noted, that communication issue can be handled by being explicit about it. 

But it's still an issue and not the only one. Before the advent of single-shot protocols, we could safely assume that 200% represents full coverage of the population. With single-shot protocols, we may be able to achieve full coverage much earlier so the threshold will no longer be constant and depend on the type of vaccines deployed. This is an issue in charts like [this one](https://pandem-ic.com/vaccine-coverage-of-the-total-population/).

A potential solution would be to construct a variable of `double-protocol equivalents` (DPE). For countries where the information is available we can derive the doses administered under single-protocols and multiply this by two to arrive at the DPE. Then we add up the remainder of the total minus the single-protocol doses, weighted by 1. If we do this then we would achieve full vaccination again at 200%. For countries where the information is not available we can just assume in the absence of other information that the vaccines administered require two shots.

An alternative to this would be to convert `total_vaccinations` into `single-protocol equivalents` (SPE). This would speak to the suggestion of @mikesherov. It would require multiplying double-protocol doses by 0.5. Then full coverage would be achieved at 100%. 

Finally, note that this is not a suggestion to drop `total_vaccinations` but rather to complement it and potentially introduce a new variable.

**Comment by edomt** [comment]
Hi @fibke 

Many thanks for writing down your thoughts on this (and apologies for the late reply!).

The equivalence of `total_vaccinations` = `people_vaccinated` is only something we apply when:

- The vaccination has just begun (< 3 weeks ago), meaning that no one in the country should have received a second dose;
- Or the country clearly reports this as the number of first doses administered, and also mentions clearly that no second dose has been administered (e.g. Ukraine currently).

Until now, we've always been able to know whether shots belonged to a single- or double-shot protocol. J&J is the only 1-dose protocol that is being used, and only in two countries (US & South Africa). South Africa only uses J&J, so that keeps things simple. And the US reports the data in a very detailed way, which lets us know clearly which doses are J&J ones vs all the other 2-dose protocols.

Since February, we've recentered our communication around `people_vaccinated`, with the increasing evidence that the first vaccine dose brings a high level of protection, and that the second dose really serves as a booster (as opposed to being absolutely necessary to be protected). As you noted, the downside of this approach is that a handful of countries (including the UAE and China) still don't report this data regularly/at all. My hope is that this information will be provided over the next few weeks.

While I like the approach you suggested of converting this information to SPE/DPE to allow for better comparison, for now I think this would introduce much more complexity than is necessary in the current "data landscape". Our (OWID's) main goal remains to make the research and data on COVID-19 vaccinations more understandable and accessible. Currently, `people_vaccinated` gives 90% of the information needed when people want to know where a country stands in terms of cumulative vaccinations. Readers can switch to looking at `total_vaccinations` if needed (e.g. UAE/China). And [this chart](https://ourworldindata.org/grapher/covid-fully-vs-one-dose-vaccine) that we added a few days ago also offers a useful comparison of dose strategies.

**Comment by ebosveld** [comment]
As an additional question, it might be good to think about how you're going to handle booster shots while other countries are still doing the primary vaccination.

**Comment by philipschellekens** [comment]
> As an additional question, it might be good to think about how you're going to handle booster shots while other countries are still doing the primary vaccination.

That's a good point. I can see 3 options: 

1/ start reporting things separately (similar to the current OWID approach of reporting at least one vaccination and fully vaccinated). Suspect this will be a preferred approach if the objective is to stick as closely as possible to the primary data as platforms like OWID like to do (@edomt: correct?).

2/ convert vaccinations into full-efficacy-equivalents. This idea is similar to the person-equivalent poverty measures of Foster and Castleman (https://cpb-us-e1.wpmucdn.com/blogs.gwu.edu/dist/5/1304/files/2018/05/Smith_Person_equivalent-ruzabw.pdf). Would control for the efficacy of all vaccines. May be impossible to implement but worth some consideration.

3/ ignore. As of now, we're mostly interested in making sure people around the world get elementary protection. Many countries are delaying second doses in order to be able to expand coverage more quickly. That may change soon as the focus shifts on the efficacy of vaccine administration particularly if variants will have it their way. Seems to be a good solution for now, but not necessarily during a later stage of the pandemic.

**Comment by edomt** [comment]
Yes, for now I think that we'd go for option 1, i.e. a new metric.

**Comment by NicolasAmbrosioni** [comment]
Hi all,

First of all, kudos to the OWID team for the amazing jobs you do!

One question (not sure I am on the right thread for this question) : I have noticed that the fully vaccinated figure for France is different between OWID and the French open source data website (https://www.data.gouv.fr/fr/datasets/donnees-relatives-aux-personnes-vaccinees-contre-la-covid-19-1/).

France has recently changed the classification from "people vaccinated with 2 doses" to "people fully vaccinated" . They made this change because "2 doses" is no longer equivalent of "fully vaccinated" because of :
- the start of vaccination with one-dose vaccines such as J&J
- for people who previously got infected by Covid-19, they will receive only one dose of a 2-dose vaccines and be considered as "fully vaccinated"
- a third dose (booster) of a 2-dose vaccines might be necessary for people who have health issues. My understanding is that this class won't be classified as "fully vaccinated" until they receive a third shot... Big question : how the French vaccination program know if someone need a third dose so that they are not classified as "fully vaccinated" after the second dose. My guess : so far, all people who have received 2 doses of a 2-dose vaccine are considered as "fully vaccinated".

As of 9th May 2021, the difference between "people vaccinated with 2 doses" and "people fully vaccinated" is getting significant (arounf 500k, i.e. arounf 1% of the French adult population) :
- people vaccinated with 2 doses : 7,768,271 (source : vacsi-fra-2021-05-10-19h05.csv on the French open source data website)
- people fully vaccinated : 8,253,826 (source : vacsi-fra-2021-05-10-19h05.csv on the French open source data website)

In comparison, the OWID figure for France as of 9th May is 7,88m people fully vaccinated.

So, how does OWID take into account the change of classification due to the start of vaccination with one-dose vaccines (such as J&J) and for people who only receive one dose of a 2-dose vaccine since they previously got infected by Covid-19?

Have a good day

Nicolas

**Comment by edomt** [comment]
Bonjour Nicolas, and thank you!

We're currently using the file `vacsi-v-fra-YYYY-MM-DD-HHhMM.csv`. It breaks down the vaccinations by vaccine name and 1st/2nd dose.

As of today, the latest data in the file is:

vaccine|n_cum_dose1|n_cum_dose2
-|-|-
Pfizer/BioNTech|12305977|6850988
Moderna|1361638|802974
Oxford/AstraZeneca|4049255|114241
Johnson&Johnson|112311|68

I'm not sure what this 68 in the last cell is supposed to represent… the J&J vaccine should never have 2nd doses.

In terms of `people_vaccinated` (at least 1 dose), we sum the first column, which gives us 17,829,181.

In terms of `people_fully_vaccinated` (full protocol), we sum the value of `n_cum_dose2` for Pfizer + Moderna + AZ and the value of `n_cum_dose1` for J&J, which gives us 7,880,514.

But I can see that indeed, in the latest version of `vacsi-fra-2021-05-10-19h05.csv`, `n_cum_complet` is given as 8,253,826 — but frankly I don't understand how they arrive at this figure based on what they show in `vacsi-v-fra`. Maybe something to raise with Etalab or SPF?

**Comment by NicolasAmbrosioni** [comment]
Merci Edouard for your reply :)

I also noticed the second doses for J&J... As Nicolas Berrod suggested on Twitter, it might be a mistake made by some vaccination centers that ticked the wrong box...

Regarding the `people_fully_vaccinated`, my understanding is that SPF sums the following figures :
1. `n_cum_dose2` for Pfizer
2. `n_cum_dose2` for Moderna
3. `n_cum_dose2` for AZ
4. `n_cum_dose1` for J&J (+maybe `n_cum_dose1` for J&J)
5. `n_cum_dose1` for people who already had Covid-19 (since, in France, it has been decided that they will only get one dose of a 2-dose vaccine and hence be considered as "fully vaccinated")
6. Number of people who gets a third dose (booster) of a 2-dose vaccines (for instance, people who have health issues). It gets tricky here because you have a risk of double count with point 1 to 3 (here above) + you can't know for sure when / if someone with health issues is going to have a third dose... So my guess is that SPF could not be bothered and consider people with health issues who already got 2 doses and are candidate for a potential third dose as "fully vaccinated". Furthermore, I am not even sure that anyone already have had a third doses in France so far.

You can find the new methodology here : https://www.santepubliquefrance.fr/presse/2021/vaccination-contre-la-covid-19-nouvelles-evolutions-des-indicateurs-disponibles-en-open-data

So the difference between OWID and SPF for the `people_fully_vaccinated` figures is point 5: `n_cum_dose1` for people who already had Covid-19 (since, in France, it has been decided that they will only get one dose of a 2-dose vaccine and hence be considered as "fully vaccinated").

This explains the 373,312 difference in the `people_fully_vaccinated` figures...

And I agree that the classification of SPF is a bit misleading because they put in place this new indicator ("fully vaccinated") in allmost all their files (of which vacsi-fra-YYYY-MM-DD-HHhmm.csv) but kept the initial indicator ("2 doses") for a couple of others (of which vacsi-v-fra-YYYY-MM-DD-HHhmm.csv). I think one of the reason they did that is that otherwise one could not calculate otherwise the number of total injections.

**Comment by StateCenterKid** [comment]
In the OWID_WRLD records, the relationship between Total_Vaccinations (doses) and people_vaccinated + people fully vaccinated is increasingly problematic.   By definition, total_vaccinations should be roughly equal to the sum of people_vaccinated and people_fully_vaccinated.   On 6 June, total_vaccinations is ~2.2B while the sum of the components is ~1.4B.  Is there any guidance on how to understand this discrepancy?

**Comment by edomt** [comment]
Hi @StateCenterKid 

This is due to some countries (notably China) not reporting the number of people vaccinated. See [here](https://twitter.com/redouad/status/1395011431641362433) for more information.

The following section of [our FAQ](https://ourworldindata.org/covid-vaccinations#frequently-asked-questions) may also be useful.

### How can we best estimate how many people have been vaccinated globally?

Because some countries — notably China, as of June 2021 — do not report breakdowns between first and second doses administered, we cannot know the exact number of people with at least 1 dose and people fully vaccinated in the world. The figures displayed on our charts for “World” only include data from countries that do publish this more precise information. Therefore, as long as some countries are only reporting partial data, these figures will underestimate the real number of people vaccinated and fully vaccinated globally.

Based on the number of doses administered, it is however possible to calculate a lower and upper bound of how many people have been vaccinated in these countries. For example, if 500 million doses have been used in China, they must have been administered to at least 250 million people (with 2 doses each), and up to 500 million people (with only 1 dose each).

We recommend citing the figure on our charts as the “confirmed number of people with at least 1 dose”.

A good proxy for estimating the “complete number of people with at least 1 dose” is:

    [confirmed number of people with at least 1 dose] + ([total doses administered in China] / 2) as a lower-bound estimate;
    [confirmed number of people with at least 1 dose] + [total doses administered in China] as a higher-bound estimate.

**Comment by SchiffS** [comment]
It would be extremely useful if we could access data on number of vaccinations versus type.  Currently I am working on an estimate of R_0 for the Delta variant, based on the current exponential growth rate in cases in the UK.  That estimate will be used to assess the vulnerability of the US to a new wave in infections.  But to do that, it is necessary to estimate the susceptible fraction of the population, which includes a fraction of the people who have been vaccinated.  For instance, the MNRA vaccines offer ca 90% protection, which means that 10% of those who have received those vaccines are susceptible.  For the J&J vaccine, the rate of protection is ca 70%, meaning 30% are susceptible.
By knowing how many shots of each type have been administered, it is possible to calculate how many more are needed to reach herd immunity, given R_0.

**Comment by edomt** [comment]
@SchiffS Are you referring to [this data](https://github.com/owid/covid-19-data/blob/master/public/data/vaccinations/vaccinations-by-manufacturer.csv) or is something missing from that?

**Comment by PierreKiwi** [comment]
Hello @edomt,
Question from a French living in NZ ;)

Regarding France, the [source](https://www.data.gouv.fr/fr/datasets/donnees-relatives-aux-personnes-vaccinees-contre-la-covid-19-1/) that you use reports 49.41M fully vaccinated people but, your [file](https://github.com/owid/covid-19-data/blob/master/public/data/vaccinations/country_data/France.csv) only counts 45.25M. I do know that there is a gap of few days, but is there an explanation for this huge difference?

For comparison, the numbers for NZ are perfectly reported (NZ numbers are way simpler to understand!!).

**Comment by edomt** [comment]
Hi @PierreKiwi  :) 

In our data, fully-vaccinated people are those who have received all doses prescribed by their vaccination regimen (e.g. 2 doses for Pfizer/BioNTech, Moderna, Oxford/AstraZeneca, etc. and 1 dose for Johnson & Johnson, CanSino, etc.).

Some countries—including France—use alternative definitions, such as having been infected with SARS-CoV-2 in the past and having received 1 dose of a two-dose regimen. We currently ignore these alternative definitions to preserve the common definition of fully vaccinated, i.e. all doses required in the vaccine regimen. This allows for an optimal comparability between countries.

**Comment by PierreKiwi** [comment]
@edomt - hi,
Thanks for your prompt response! I found the explanation in your [FAQ][0] as well, so sorry for disturbing you. However, it offers me a good rebound so let's grab it :).

Your approach makes sense as it is def. a thorny problem. You want to offer data which is reliable and meaningful. Nevertheless I would argue that OWID results can be misleading for two reasons: OWID does not seem to apply the same logic to each country and eligibility age/demographics are not the same between countries. Let me break them down.

- For the UAE, your [source][1] indicates 96% and 86% (one dose - fully vaccinated respectively). Despite my efforts, I haven't been able to found the exact breakdown on the vaccines they have administered. On top of that, the number of  doses given does not really match their overall population (10m people but 20+m doses). OWID seems to report the official % as they are (95% and 85% respectively, small difference due to a lag in reporting I guess).

- For Canada, your [source][2] indicates 77% and 72.5%. Even if Canada has mixed vaccines for several millions people (almost 4m, >10% of the total, cf. [here][3]), OWID still report the same %. I was kind of expecting some differences here. Maybe Canada has perfectly mixed the vaccines, but that sounds unlikely because they seemed to have used multiple combinations. On top of that, Quebec uses the same definition as France for fully vaccinated people (confirmed Covid + 1 dose) (cf. [here][5]).

- For France, which has indeed a different definition of what fully vaccinated means, OWID uses the exact breakdown per vaccine to calculate the % of fully vaccinated people rather than accepting the official numbers (as you do with the UAE and Canada). That means OWID numbers are 75%/67% while official numbers are 75%/73% (I used 67,467,000 for the total population of France based on [wiki][4]). Sounds like a penalty on France because its numbers are too detailed :).

At this stage, would it not be way easier for OWID to simply use the official data of each country (especially when they are reliable) for both one dose and fully vaccinated rather than trying to follow the official vaccine regimen (which is already hijacked in some ways (boosters, mixing, one dose + covid = fully vaccinated))?

Regarding my second point. Because demographics between countries are very different (young African country, old Europeans ones) and there is no consistency about eligibilty (some countries vaccinate from 3 like the UAE (TIL!) while many allow it from 12+ (France, Canada, NZ), and others from 18+), what about reporting % based on the eligible population rather than the overall one?

From an OWID point of view, I do understand that it is easier to use the total population as denominator because some countries do not report more detailed data. But I found the % not really representative of the vaccinal cover. For example, using their eligible population, Canada, France, and NZ jump quite significantly:

- Canada: 77 -> 88% (one dose) and 72 -> 82% (12+ fully vaccinated).
- France: 75% -> 88% (one dose) and 73% -> 85% (12+ fully vaccinated).
- NZ: 70% -> 83% (one dose) and 52% -> 62% (12+ fully vaccinated).
- Can't calculate for the UAE because I haven't found the breakdown by age.

Regards

---
[0]: https://ourworldindata.org/covid-vaccinations#frequently-asked-questions
[1]: https://fcsc.gov.ae/en-us/Pages/Covid19/UAE-Covid-19-Updates.aspx
[2]: https://covid19tracker.ca/vaccinationtracker.html
[3]: https://health-infobase.canada.ca/covid-19/vaccination-coverage/#a6
[4]: https://en.wikipedia.org/wiki/Demographics_of_France
[5]: https://health-infobase.canada.ca/covid-19/vaccination-coverage/technical-notes.html#a5

**Comment by edomt** [comment]
Hi @PierreKiwi 

On the contrary, we do apply the same methodology to every country! Or at least as much as we can :)

- For the UAE, we don't use the % as published by the source (we never do — we always collect absolute numbers, and then divide them by our own denominators). The complete method can be seen [here](https://github.com/owid/covid-19-data/blob/master/scripts/src/cowidev/vax/incremental/united_arab_emirates.py), but basically we use the "N doses per 100 people" metric on the official dashboard to find out the population estimate used in the official data. We also calculate the ratio of 1st-to-2nd doses based on the two reported percentages and the fact that the UAE only uses 2-dose vaccines. And based on this, we calculate the absolute number of 1st and 2nd doses used in the country.

- For Canada (and potentially other countries), we can indeed be limited by what the country reports. If the country uses an alternative definition of "fully vaccinated" (such as past infection + 1 dose) but only publishes a "fully vaccinated" metric instead of detailed data, then we cannot recalculate the correct version of the metric. Indeed, this means that for France, we're able to calculate it — but I wouldn't call that a penalty, rather I'd say the data is more accurate. These country-specific definitions are changing all the time, and are based on very little scientific evidence (e.g. considering that someone who was infected with COVID in March 2020 and has had one dose of vaccine in March 2021 is "fully vaccinated"  is really stretching the definition). More generally, the relevance of "people fully vaccinated" is dropping very quickly, with booster doses being administered and some countries now deciding that the original definition of a full vaccination wasn't the right one (e.g. people vaccinated with J&J asked to get a second dose).

Regarding your two last suggestions, beyond the fact that this wouldn't be a good idea in terms of comparability between countries, it would also be extremely difficult for us to change either our denominators, or our method of data collection. This would involve revising scripts and historical data for ~200 countries.

**Comment by willgilks** [comment]
Hi @edomt,

I'm trying to calculate the number of daily vaccinations of each dose, using people_vaccinated, people_fully_vaccinated and total_vaccinations. I'm approximating missing values, then calculating a daily rate from the differnce between consecutive cumulative values. I have data on the proportion of vaccines delivered which are one- or two-shot vaccines from another source.

I was wondering if you were able to help with the following:

Number of booster shots should be total_vaccinations minus people_vaccinated. 

The number of people recieving one shot of a one-shot only vaccine is people_fully_vaccinated multiplied by the proportion of vaccines in that country which are one-shot only.

The number of people recieving the second shot of a two-shot vaccine is people_fully_vaccinated multiplied by the proportion of vaccines in that country which are two-shot.

The number of people receiving the first shot of a two-shot vaccine is people_vaccinated minus the number of people recieving the only shot of a one-shot vaccine. This generates negative values though so I think my calculation of those receiving only shot of one-shot is incorrect.

I've read the definitions information on https://github.com/owid/covid-19-data/tree/master/public/data/vaccinations I understand that public health data can be difficult to standardise, especially under conditions of global epidemic. Any help would be greatly appreciated.

**Comment by edomt** [comment]
Hi @willgilks 

> Number of booster shots should be total_vaccinations minus people_vaccinated

This is unfortunately invalid: `people_vaccinated` is the number of first doses, so `total_vaccinations – people_vaccinated` gives you the number of second doses + booster doses.

The best solution I can see would be:

- `people_vaccinated` = first doses. Note that, however, some of these will be first doses of J&J or CanSino, i.e. first doses that make up a full vaccination.
- `total_boosters` = booster doses. Note that these aren't always 3rd doses: they could be a 2nd dose of a 1-dose vaccine, or a 3rd (or even 4th in Turkey now) dose of a 2-dose vaccine.
- `total_vaccinations – people_vaccinated – total_boosters` = non-first & non-booster doses. These are basically doses that "complete the protocol" after the first dose. Most of them will be 2nd doses of 2-dose vaccines, but you'll have a few 3rd doses of 3-dose vaccines (e.g. in Cuba)

**Comment by leodiegues** [comment]
Hi all, 

Don't know if this is the right place to ask but I've been having issues with the `people_fully_vaccinated_per_hundred` column.

The numbers I get dividing the `people_fully_vaccinated` by `population` columns for the USA differs from OWID's table, starting from 2021-01-29. I don't get the same issues doing the same for other countries.

Previous to 2021-01-29 most of the differences are insignificant and I believe could be related to rounding issues. However the gap between my calculation and the OWID's `people_fully_vaccinated_per_hundred` column grows to almost 1pp distance at the latest date available.

US examples:

Population as is on `population` column: 332915074

- 2021-01-29:
    - `people_fully_vaccinated`: 4780888
    - `people_fully_vaccinated_per_hundred`: 1.42
    - expected result: 1.44
- 2021-05-01:
    - `people_fully_vaccinated`: 103422555
    - `people_fully_vaccinated_per_hundred`: 30.75
    - expected result: 31.07
- 2021-10-19:
    - `people_fully_vaccinated`:  189292559
    - `people_fully_vaccinated_per_hundred`: 56.28
    - expected result: 56.86

Am I missing any methodological steps here?

**Comment by PierreKiwi** [comment]
@edomt 

Thanks a lot for taking the time to answer to my comments.

I was not implying that OWID should drop the _relative to population_ aspect, but eventually try to add the _relative to eligible population_ option. A lot of work for sure :).

Regarding the differences between countries, thanks a lot for all the details. I now understand OWID approach and will look at the numbers with a different mindset.

**Comment by philipschellekens** [comment]
I agree with Edouard that an indicator that constrains the denominator to the eligible population will be hard to construct. While it may help assessing the vaccination performance of countries assuming eligibility criteria are common across countries, a big drawback is that it runs the risk of conveying the notion that the non-eligible group doesn’t matter in measuring vaccination coverage.

**Comment by edomt** [comment]
> Hi all,
> 
> Don't know if this is the right place to ask but I've been having issues with the `people_fully_vaccinated_per_hundred` column.

Hi @leonardodiegues - see #1983

**Comment by leodiegues** [comment]
@edomt Thank you!

**Comment by willgilks** [comment]
Hi, does the new_vaccinations columns include boosters? (Apologies if it's stated in the manual, just require confirmation)

**Reply by edomt** [reply]
Hi @willgilks - yes, `new_vaccinations` includes all doses counted individually (incl. boosters).

**Reply by willgilks** [reply]
Thank you :)

**Comment by willgilks** [comment]
if i deaccumulate _people_vaccinated_, _people_fully_vaccinated_, _total_boosters_, _total_vaccinations_, then the sum of the first three is greater than _total_vaccinations_ on 12,030 days (91 countries involved), corresponding to 81,605,564 doses administered. In an attempt to deal with this, I maintain the _total_boosters_ value but reduce _people_vaccinated_ and _people_fully_vaccinated_ proportionally to fit with the remaining (tv-_total_boosters_). Unfortunately, there's still 88 days where _total_boosters_ per day exceeds _total_vaccinations_ per day (by a total of 2,419,282 doses, 14 countries involved). lmk if my logic is wrong here.

If i change _total_vaccinations_ on these days to be the higher _total_boosters_ value, then of course the cumulative total _total_vaccinations_ increases. Instead, I could replace the 'bad' daily value with zero or interpolate it. Either way, what remains then is to distribute the unaccounted for cumulative doses among all days providing this doesn't increase their daily vaccinations to greater than the total vaccinations.

This is very fiddly, and means that all historical data points are being changed every day. A less laborious solution might be to loess-smooth on the non-cumulative to remove likely artifacts (possibly omitting days from the model where the subgroup total exceeds _total_vaccinations_), then accumulate, then adjust by the reported cumulative total.

So my question is, how do I deal with the 88 days where _total_boosters_ per day is greater than _total_vaccinations_ per day ? Is there a reasonably possibility that this can be corrected in the available data? attaching csv of problem days.

Thanks for all your hard work, hugely apprieciated.

[owid_vacns_exceed_tv_20221027.csv](https://github.com/owid/covid-19-data/files/9877795/owid_vacns_exceed_tv_20221027.csv)

**Reply by lucasrodes** [reply]
Hi @willgilks! 
Thanks for your message.

Metrics people_* count people while the others count doses, so it is technically not correct to sum these metrics.

Also, people_fully_vaccinated counts people with one dose of single shot vaccines (janssen or cansino), so these are counted twice in your computations.

You can read more about the metric definitions [here](https://github.com/owid/covid-19-data/blob/master/public/data/README.md).


I hope that this helps,
Best,

**Reply by willgilks** [reply]
Hi @lucasrodes, thanks, got it. What about daily _total_boosters_ being greater than daily _total_vaccinations_? De-accumulated values would of course appear like this if the _total_boosters_ data starts at e.g 20% of population, but there are other cases. see ARG, MAC, RUS, SUR, SVN, VNM
